lib/matplotlib/textpath.py

```diff
@@ -68,11 +67,12 @@ def _get_char_id_ps(self, font, ccode):
         return char_id
 
     def glyph_to_path(self, font, currx=0.):
-        """
-        convert the ft2font glyph to vertices and codes.
-        """
+        """Convert the *font*'s current glyph to a (vertices, codes) pair."""
         verts, codes = font.get_path()
         if currx != 0.0:
+            cbook._warn_deprecated(
+                "3.1", name="currx", obj_type="argument",
+                alternative="Manually translate the path vertices.")
```

Comment:
currx is not used in the codebase and that's really trivial for the caller to do it

Reply:
I really dislike slipping in deprecations in a commit that says it's about trivial changes to docstrings.

Reply:
Sorry about that, reverted.